sdcm/ec2_client.py

```diff
@@ -240,12 +242,13 @@ def get_instance(self, instance_id):
 
     @retrying(n=5, sleep_time=10, allowed_exceptions=(ClientError,),
               message="Waiting for instance is available")
-    def add_tags(self, instance_id, tags=None):
+    def add_tags(self, instance_ids, tags=None):
```

Comment:
would an instance have more than one ID?

Reply:
or it should be rather `instances_ids`?

Reply:
we can add tags to multiple instances at the same time (we can also tag other resources as the same time)